src/AwsPinpointClient.php

```diff
@@ -41,7 +41,7 @@ public function send(AwsPinpointSmsMessage $message)
                         'SMSMessage' => [
                             'Body' => $message->body,
                             'MessageType' => $message->messageType,
-                            'SenderId' => config('aws.Pinpoint.sender_id'),
+                            'SenderId' =>  $message->setSenderId,
```

Comment:
Can you make it so that it only uses the senderId if it is set on the `$message`? It should use the value defined in the config if none is set on the message or else it will have no senderId if you don't set it using `->setSenderId($notifiable->senderId)` and it will fail.

Also, it should be `$message->senderId` instead of `$message->setSenderId`.